commune_app/all_models/votes.py

```diff
@@ -20,3 +20,5 @@ def create_new_vote(voting_user, voted_chore, vote_bool):
             if num_of_yes_votes > num_of_no_votes:
                 voted_chore.passed = True
                 voted_chore.save()
+    def vote(self, chore_id, decision):
```

Comment:
./commune_app/all_models/votes.py:23:5: E301 expected 1 blank line, found 0
```suggestion

    def vote(self, chore_id, decision):
```